viewer/src/public/migration/Cognite3DViewer.ts

```diff
@@ -473,11 +473,16 @@ export class Cognite3DViewer {
     if (options.onComplete) {
       throw new NotSupportedInMigrationWrapperError('onComplete is not supported');
     }
-    const { modelId, revisionId } = options;
-    const cadNode = await this._revealManager.addModel('cad', {
-      modelId,
-      revisionId
-    });
+    const { modelId, revisionId, geometryFilter } = options;
+    const cadNode = await this._revealManager.addModel(
+      'cad',
+      {
+        modelId,
+        revisionId
+      },
+      undefined,
```

Comment:
any way we can void sending in undefined here?

Reply:
Fixed by creating a `AddCadModelOptions` type in `RevealManager.